src/common/core/utils.py

```diff
@@ -84,6 +89,23 @@ def get_version_info() -> VersionInfo:
     return version_json
 
 
+def get_version_number() -> str:
+    """Returns the version number of the current deployment"""
+    manifest_versions = get_versions_from_manifest()
+    return manifest_versions.get(".", UNKNOWN)
+
+
+@lru_cache()
+def get_versions_from_manifest() -> VersionManifest:
+    """Reads the version info from the manifest file"""
```

Comment:
nit: I prefer [PEP 257](https://peps.python.org/pep-0257/) imperative style for docstrings:

```suggestion
    """Read the version info from the manifest file"""
```

Reply:
I always prefer imperative verb form though other docstrings are 3rd-person, so I chose consistency. Do you think we should update only these now, and the rest later?

Reply:
I think the PEP makes a lot of sense and would like to add it to team conventions for code in the future. I'd rather the new code to be compliant to it, if you agree